java/src/lib.rs

```diff
@@ -206,6 +206,17 @@ pub extern "system" fn Java_glide_ffi_resolvers_ScriptResolver_dropScript<'local
     _class: JClass<'local>,
     hash: JString,
 ) {
-    let hash_str: String = env.get_string(&hash).unwrap().into();
-    glide_core::scripts_container::remove_script(&hash_str);
+    handle_panics(
+        move || {
+            fn drop_script(env: &mut JNIEnv<'_>, hash: JString) -> Result<(), FFIError> {
+                let hash_str: String = env.get_string(&hash)?.into();
+                glide_core::scripts_container::remove_script(&hash_str);
+                Ok(())
+            }
+            let result = drop_script(&mut env, hash);
```

Comment:
In theory, this method can `panic` (see its impl) - does this means we will enter an infinite loop?

Reply:
Same answer as above. No recursion is used here.